<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QMainWindow" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>210</y>
      <width>281</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Signin</string>
    </property>
    <widget class="QPushButton" name="pushbutton_logon">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>110</y>
       <width>115</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>198</width>
       <height>23</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_username">
      <item>
       <widget class="QLabel" name="label_Username">
        <property name="text">
         <string>Username</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_Username"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>196</width>
       <height>23</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_password">
      <item>
       <widget class="QLabel" name="label_Password">
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_Password">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="label_picture">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>321</width>
      <height>179</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>370</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>[+]Status</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEdit_Username</tabstop>
  <tabstop>lineEdit_Password</tabstop>
  <tabstop>pushbutton_logon</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>lineEdit_username_2</tabstop>
  <tabstop>lineEdit_password_2</tabstop>
  <tabstop>lineEdit_password</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>lineEdit_username</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
